release.config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @type {import('semantic-release').GlobalConfig}
+ */
+module.exports = {
+  branches: ['main'],
+  plugins: [
+    '@semantic-release/commit-analyzer',
+    '@semantic-release/release-notes-generator',
+    '@semantic-release/changelog',
+    '@semantic-release/npm',
+    [
+      '@semantic-release/git',
+      {
+        assets: ['CHANGELOG.md', 'package.json', 'package-lock.json'],
+        message:
+          // eslint-disable-next-line no-template-curly-in-string
+          'chore(release): ${nextRelease.version} \n\n${nextRelease.notes}',
+      },
+    ],
+    '@semantic-release/github',
+  ],
```

Comment:
I wrote this with reference to the guides introduced in [@semantic-release/changelog](https://github.com/semantic-release/changelog?tab=readme-ov-file#usage) and [@semantic-release/git](https://github.com/semantic-release/git). There're no special settings.

Reply:
The Usage docs for @semantic-release/git include the keyword [skip ci] in the commit message, but it doesn't seem to be included here.
Is there an intention to remove this?

My guess is that you have decided that this is unnecessary because multiple branches do not run in parallel due to trunk-based development.

Reply:
exactly. I believe this repo doesn't need `[skip ci]` option.

Reply:
I see! Thanks!